docs/index.rst

```diff
@@ -21,29 +20,30 @@ Why Use Crud
 
 * CRUD will dynamically add the actions to your controller so you don't have to re-implement them over and over again.
 
-* CRUD does not have the same limitations as CakePHP's own scaffolding, which is "my way or the highway." CRUD allows you to hook into all stages of a request, only building the controller code needed specifically for your business logic, outsourcing all the heavy boiler-plating to CRUD.
+* CRUD allows you to hook into all stages of a request, only building the controller code needed specifically for your business logic, outsourcing all the heavy boiler-plating to CRUD.
```

Comment:
General comment on the use of "CRUD" vs "Crud":

- I personally prefer "Crud" for both readability and the fact that it matches the code
- either way, consistent use throughout the docs would be a win IMO